### ISSUE TITLE
Make webfont easier to change for non-CFPB use cases

### ISSUE THREAD

**Issue by himedlooff** [cfpb/cf-core#23]
Currently non-CFPB projects have to deal with ripping out Avenir. We should make it easier to replace it by creating variables.

How does the following look to you all?

**cf-vars.less**

``` css
...
@webfont_declarations: "/static/css/licensed-fonts.css";
@webfont_regular: "'AvenirNextLTW01-Regular', Arial, sans-serif"
// Then vars for each version of Avenir.
...
```

**cf-base.less**

``` css
...
.webfont-regular() {
    font-family: @{webfont_regular};
    font-style: normal;
    font-weight: normal;

    & em,
    & i {
        .webfont-italic();
    }

    & strong,
    & b {
        .webfont-demi();
    }
}
// Then one for each version of Avenir.
...
```

**cf-core.less**

``` css
...
@import "@{webfont_declarations}";
...
```

So by default a CFPB project shouldn't have to do anything different. Projects that want to use other font families can point to their own file containing their own `@font-face rules`. They'd do it like this:

**main.less**

``` css
@import "cf.less"; // Assume that this is a concatenated version of Capital Framework
@webfont_declarations: "/static/css/my-own-font-face-rules.css";
@webfont_regular: "'MyFontFamilyNameForRegular', sans-serif";
// Etc...
```

CC: @ascott1

**Comment by Scotchester** [comment]
Makes sense to me!

**Comment by ascott1** [comment]
Yes, that makes a lot of sense! Nicely done @himedlooff.

**Comment by virginiacc** [comment]
This looks great, @himedlooff!

**Comment by KimberlyMunoz** [comment]
Closed by #36
